lib/ecto/adapter/schema.ex

```diff
@@ -87,6 +87,6 @@ defmodule Ecto.Adapter.Schema do
   in case there is no record matching the given filters,
   `{:error, :stale}` is returned.
   """
-  @callback delete(adapter_meta, schema_meta, filters, options) ::
+  @callback delete(adapter_meta, schema_meta, filters, returning, options) ::
```

Comment:
I think we need to have both arities.

Reply:
Older versions will get the warning but still work, that's good enough to me. :)

Reply:
ah thanks I didn't realize that